README.md

```diff
@@ -24,6 +24,52 @@ See also:
 
 Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF). If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how Jaeger plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2017/09/13/cncf-hosts-jaeger/).
 
+## Features
+
+### High Scalability
+
+Jaeger backend is designed to have no single points of failure and to scale with the business needs.
+For example, any given Jaeger installation at Uber is typically processing several billions of spans per day.
```

Comment:
Do you wanna link to http://jaeger.readthedocs.io/en/latest/deployment/#collectors or describe how it's scalable somewhere in the docs?

Reply:
we can change it later once we have the docs. The link to collectors seems a bit arbitrary, as the no-SPOF argument must apply to the whole stack.